recipes/glfw/all/conandata.yml

```
@@ -21,6 +21,36 @@ sources:
     url: "https://github.com/glfw/glfw/releases/download/3.3.2/glfw-3.3.2.zip"
     sha256: "08a33a512f29d7dbf78eab39bd7858576adcc95228c9efe8e4bc5f0f3261efc7"
 patches:
+  "3.3.8":
+    - patch_file: "patches/3.3.5-0001-Wayland-Remove-extra-cmake-modules-dependency.patch"
+      patch_description: "Remove dependency on extra-cmake-modules to fix generation of Wayland Protocols"
+      patch_type: "upstream"
```

```suggestion
      patch_type: "official"
```
Upstream is an invalid name:

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conandata_yml_format.md#patch_type

Please, don't forget to update others.

---

Thanks!